<launch>

  <arg name="launch_prefix" default=" " />

    <node pkg="detection_hg" type="detect" name="detect" output="screen" launch-prefix="$(arg launch_prefix)">

    	<!-- subscribed topics -->
        <remap from="~input_image" to="/camera/rgb/image_rect_color"/>
  	<remap from="~input_bbox" to="/darknet_ros/bounding_boxes"/>

    <rosparam subst_value="true">
    max_keypoints: 20
    pkg_dir: $(find detection_hg)
    nn_modle_weights: model.t7
    vis_out: true
    pub_hms: true
    </rosparam>
    </node>

</launch>
